hudi-common/src/main/java/org/apache/hudi/common/table/log/block/HoodieDeleteBlock.java

```diff
@@ -142,11 +142,18 @@ private byte[] serializeV3() throws IOException {
     HoodieDeleteRecordList.Builder recordListBuilder = HOODIE_DELETE_RECORD_LIST_BUILDER_STUB.get();
     HoodieDeleteRecord.Builder recordBuilder = HOODIE_DELETE_RECORD_BUILDER_STUB.get();
     List<HoodieDeleteRecord> deleteRecordList = Arrays.stream(getRecordsToDelete())
-        .map(record -> HoodieDeleteRecord.newBuilder(recordBuilder)
+        .map(record -> {
+          List<Comparable> orderingVals = record.getOrderingValue() instanceof Comparables
```

Comment:
not a fan of `instanceof` per record, after this patch, the `record.getOrderingValue()` should always be `Comparables` right? If this is true, could we switch it's return type to just `Comparables`?